ros-integration/src/lib.rs

```diff
@@ -43,11 +43,13 @@ use robonomics_runtime::{
     robonomics::*, RobonomicsCall, Nonce, Runtime
 };
 
-mod ros;
 mod msg;
 mod ipfs;
 mod rosbag_player;
 
+#[macro_use]
+extern crate rosrust;
```

Comment:
Isn't needed in most cases, we use edition 2018:
https://doc.rust-lang.org/edition-guide/rust-2018/module-system/path-clarity.html